webservices/resources/reports.py

```diff
@@ -105,11 +134,18 @@ def build_query(self, committee_id=None, committee_type=None, **kwargs):
         if kwargs.get('is_amended') is not None:
             query = query.filter(reports_class.is_amended == kwargs['is_amended'])
 
+        query = filters.filter_range(query, kwargs, self.filter_range_fields)
         return query, reports_class, reports_schema
 
     def _resolve_committee_type(self, committee_id=None, committee_type=None, **kwargs):
-        if committee_id is not None:
-            query = models.CommitteeHistory.query.filter_by(committee_id=committee_id)
+        #if kwargs.get('committee_id') is not None and len(kwargs.get('committee_id')) > 0:
+        if committee_id is not None or kwargs.get('committee_ids'):
+            comm_id = (
+                committee_id
+                if committee_id is not None
+                else kwargs.get('committee_ids')[0]
```

Comment:
My thinking what that getting no is much clearer than getting some results. You will assume you are getting all your results if it looks like it is working. 

I do think you have a good point about the url structure adding some clarity and expectations for data users. We will probably want to add something in the docs about all the committees being the same type to. 

I would be cool with adding multiple committees of the same type if we don't check the type here, and use [the url check](https://github.com/18F/openFEC/pull/1864/files/158a0b19d39e5b62aefb987497a3494bd0187bbb#diff-3e2e29f1fb0dc0c4c17dc617d5a140c8R154) to determine the type.

I merged this, so any changes would need to be a follow up PR